src/app.scss

```diff
@@ -66,8 +64,13 @@
     }
   }
 
+  .appHeader, .logo {
+    top: unset;
+    z-index: 998;
+  }
+
   .mobileMenuContainer {
-    z-index: 998; // stay below modal screen
+    z-index: 997; // stay below modal screen
```

Comment:
just curious:  why decrease by one?

Reply:
I guess it was a left over from some experiment I've done. Changed it back.